styles/main.css

```diff
@@ -57,13 +61,17 @@ header{
 header nav{
 	float:right;
 }
+header ul {
+	margin: 0;
+	padding: 0;
+}
 #page-copyright{
 	color:#ffaa00;
 	text-transform: uppercase;
 	font-size: 3px;
 }
 figure img{
-	box-shadow: 2px 2px 0px 3px rgba(0, 0, 0, 0.2);
+	box-shadow: 2px 2px 3px rgba(0, 0, 0, 0.5);
```

Comment:
Just need three numbers (x offset, y offset, and blur radius)
Incorrect transparency